### PR TITLE
Page desiner

### PR DESCRIPTION
**Pull request by karolina-rachuta**

### CODE REVIEW

File: cartridges/app_custom_hw/cartridge/client/default/scss/experience/components/commerce_assets/youtubeBanner.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.youtubeBanner__container {
```
Comment (kalipiev) [inline cartridges/app_custom_hw/cartridge/client/default/scss/experience/components/commerce_assets/youtubeBanner.scss:1]: BS has utility classes for all of these besides gap: 8px, which I guess might be replaced with margin left/right as well.

File: cartridges/app_custom_hw/cartridge/client/default/js/profile.js
Comment (kalipiev) [inline cartridges/app_custom_hw/cartridge/client/default/js/profile.js:1]: Why do we need this empty file?

File: cartridges/app_custom_hw/cartridge/experience/components/commerce_assets/banner.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+var Template = require('dw/util/Template');
+var HashMap = require('dw/util/HashMap');
+/**
+ * Render logic for the component.
+ */
+module.exports.render = function (context) {
+   var model = new HashMap();
+   // add parameters to model as required for rendering based on the given context (dw.experience.ComponentScriptContext):
+   // * context.component (dw.experience.Component)
```
Comment (kalipiev) [inline cartridges/app_custom_hw/cartridge/experience/components/commerce_assets/banner.js:10]: Please remove this commented code if not required.

File: cartridges/app_custom_hw/cartridge/static/default/css/experience/components/commerce_assets/youtubeBanner.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.youtubeBanner__container {
```
Comment (kalipiev) [inline cartridges/app_custom_hw/cartridge/static/default/css/experience/components/commerce_assets/youtubeBanner.css:1]: Same here, you do have BS classes for all those. 
Also please mind that you don't need to write any property multiple times. Webpack will generate all that in the output css.

File: cartridges/app_custom_hw/cartridge/static/default/js/profile.js
Comment (kalipiev) [inline cartridges/app_custom_hw/cartridge/static/default/js/profile.js:1]: Please don't commit the generated files (in /static/ folder)

File: cartridges/app_custom_hw/cartridge/templates/default/experience/components/commerce_assets/banner.isml
Comment (kalipiev) [inline cartridges/app_custom_hw/cartridge/templates/default/experience/components/commerce_assets/banner.isml:1]: Hardcoded text

File: cartridges/app_custom_hw/cartridge/client/default/scss/experience/components/commerce_assets/youtubeBanner.scss (hunk 1 above)
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/client/default/scss/experience/components/commerce_assets/youtubeBanner.scss:1]: ok, I will use BS classes. I misunderstood, I will start using BS

File: cartridges/app_custom_hw/cartridge/client/default/js/profile.js
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/client/default/js/profile.js:1]: when I added this then I think finally npm run compile:scss worked, before I got some errors - I might be wrong though. I added one file here (profile.js) and one file in static folder (js->profile.js), when I ran npm run compile:js && npm run compile:scss && npm run compile:fonts then some info shown up in static file (js->profile.js)

File: cartridges/app_custom_hw/cartridge/experience/components/commerce_assets/banner.js (hunk 2 above)
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/experience/components/commerce_assets/banner.js:10]: ok

File: cartridges/app_custom_hw/cartridge/static/default/css/experience/components/commerce_assets/youtubeBanner.css (hunk 3 above)
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/static/default/css/experience/components/commerce_assets/youtubeBanner.css:1]: I didn't write anything in css file, only in scss, all the info here has shown up when I did npm run compile:scss

File: cartridges/app_custom_hw/cartridge/static/default/js/profile.js
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/static/default/js/profile.js:1]: ok

File: cartridges/app_custom_hw/cartridge/templates/default/experience/components/commerce_assets/banner.isml
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/templates/default/experience/components/commerce_assets/banner.isml:1]: ok

File: cartridges/app_custom_hw/cartridge/client/default/js/profile.js
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/client/default/js/profile.js:1]: I deleted it now and everything works

File: cartridges/app_custom_hw/cartridge/client/default/scss/experience/components/commerce_assets/youtubeBanner.scss (hunk 1 above)
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/client/default/scss/experience/components/commerce_assets/youtubeBanner.scss]: fixed

File: cartridges/app_custom_hw/cartridge/static/default/css/experience/components/commerce_assets/youtubeBanner.css (hunk 3 above)
Comment (karolina-rachuta) [reply cartridges/app_custom_hw/cartridge/static/default/css/experience/components/commerce_assets/youtubeBanner.css]: I deleted css, scss files and added BS classes

**Pull request merged**